### ISSUE TITLE
Include LICENSE file in the installed node_modules

### ISSUE THREAD

**Issue by MarkAshraf96** [rehypejs/rehype-minify#51]
### Initial checklist

- [X] I read the [support docs](https://github.com/rehypejs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/rehypejs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/rehypejs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Arehypejs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Problem

Including the license file in the installed node_modules, makes the packages compatible with license checkers and attribution file generators, that a lot of developers need for their work.

### Solution

Add the license file in the package.json to the 'files' field of each package.

### Alternatives

Nothing on my mind, that can replace that.

**Comment by wooorm** [comment]
License checkers should be able to look at the package.json? Do you use tools that don’t do that?

**Comment by wooorm** [comment]
I don’t think your solution works. If we link from `files` to `../../license`, I’m quite sure that no license file will be packaged?

**Comment by remcohaszing** [comment]
We could either copy the license in the build scripts, or just copy the license into all packages once. We already include the license in packages in other monorepos, such as [remark](https://github.com/remarkjs/remark).

**Comment by MarkAshraf96** [comment]
> License checkers should be able to look at the package.json? Do you use tools that don’t do that?

They are able to detect the license type, but they are not able to detect the license text, which is required for creating copyrights/attribution files.



> I don’t think your solution works. If we link from `files` to `../../license`, I’m quite sure that no license file will be packaged?

To be honest, I am not quiet sure that it would work or not. Another solution that I just bumped into in another monorepo, is just copying the license into all packages.

**Comment by MarkAshraf96** [comment]
> We could either copy the license in the build scripts, or just copy the license into all packages once. We already include the license in packages in other monorepos, such as [remark](https://github.com/remarkjs/remark).

And [micromark](https://github.com/micromark/micromark/tree/main) also, I believe you did the same.

**Comment by wooorm** [comment]
[released](https://github.com/rehypejs/rehype-minify/releases/tag/7.0.1)!

**Comment by MarkAshraf96** [comment]
> [released](https://github.com/rehypejs/rehype-minify/releases/tag/7.0.1)!

Thank you a lot, another thing because I was looking also into doing the same in other repos that you are maintaining. Do you think it's good if you add the license file also to the installed packages?

**Comment by wooorm** [comment]
Sorry, I don’t understand your question. Can you rephrase please

**Comment by MarkAshraf96** [comment]
Yes sure,
I mean including the license files in the package distribution.
I believe it's also considered as a best practice.

**Comment by wooorm** [comment]
That’s what I just did?

**Comment by MarkAshraf96** [comment]
My fault sorry, I didn't notice that they are already included, and the changes you did in the pipeline-package.js.
